Blish HUD/GameServices/ArcDpsService.cs

```diff
@@ -107,8 +118,37 @@ public class ArcDpsService : GameService {
             _stopwatch.Start();
         }
 
+        /// <summary>
+        /// Starts the socket listener for the arc dps bridge.
+        /// </summary>
         private void Start(object sender, ValueEventArgs<uint> value) {
-            if (this.Loaded) _server.Start(new IPEndPoint(IPAddress.Loopback, GetPort(value.Value)));
+            this.Start(value.Value);
+        }
+
+        /// <summary>
+        /// Starts the socket listener for the arc dps bridge.
+        /// </summary>
+        private void Start(uint processId) {
+            if (this.Loaded) {
+                _server.Start(new IPEndPoint(IPAddress.Loopback, GetPort(processId)));
+            }
+        }
+
+        /// <summary>
+        /// Stops the socket listener for the arc dps bridge.
+        /// </summary>
+        private void Stop() {
+            if (this.Loaded) {
+                this._server.Stop();
+            }
+        }
+
+        /// <summary>
+        /// Restarts the socket listener for the arc dps bridge.
+        /// </summary>
+        public void Restart() {
```

Comment:
Is this expected to be used outside of Core?  Who are the expected callers?

Reply:
I added that in case it does actually disconnect at some point and a module dev wants to restart it immediately

Reply:
I would prefer that we do not expose this - I don't want modules calling this.  If a module could detect failure, then it seems reasonable to me that Core should be able to as well.  👍

Reply:
I have removed the Restart and Stop (0 references) functions